### PR TITLE
MM-17350 - remove extra border

### PR DESCRIPTION
**Pull request by ali-farooq0**
#### Summary
Removes extra border on backstage settings page

#### Ticket Link
Fixes https://mattermost.atlassian.net/browse/MM-17350

#### Related Pull Requests
None

#### Before
image

image

### After
image

image

### CODE REVIEW

File: components/backstage/components/backstage_category.jsx [hunk 1]
```diff
@@ -49,6 +49,7 @@ export default class BackstageCategory extends React.Component {
                 <Route
                     path={link}
                     render={() => (
+                        children && children.length > 0 &&
```
Comment (hmhealey) [inline components/backstage/components/backstage_category.jsx]: Could this go outside of the `Route` component entirely? That way, it wouldn't render anything for the children if there aren't any provided

Comment (ali-farooq0) [reply components/backstage/components/backstage_category.jsx]: Oh yeah good call, not sure how i missed that :P

**Comment by ali-farooq0** [comment]
uh oh, was going to add the `QA` label once Dev Review was complete.

@hmhealey fyi -- our new process requires us to get QA approval before merging new changes into master. 
more context: https://community.mattermost.com/private-core/pl/c811ue6r5jbg58x9unqoctq8eo

**Comment by ali-farooq0** [comment]
@lindalumitchell this requires QA. Apologies for merging it in. I forgot to put on the `QA` label. I can revert the change if that helps with the process.

**Comment by lindalumitchell** [comment]
Thanks @ali-farooq0, that's okay. We can just test this one post-merge. Thanks for checking!

**Comment by ali-farooq0** [comment]
> Thanks @ali-farooq0, that's okay. We can just test this one post-merge. Thanks for checking!

Excellent! Does it make sense to hold off the cherry-pick (5.14) till the QA is done?
https://github.com/mattermost/mattermost-webapp/pull/3292

**Comment by lindalumitchell** [comment]
Oh thanks @ali-farooq0, no need to wait before cherry-picking. Go for it.

**Comment by ali-farooq0** [comment]
> Oh thanks @ali-farooq0, no need to wait before cherry-picking. Go for it.
👍

**Pull request merged**